### PR TITLE
engine: add changelog for 20.10.12

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@justincormack @chris-crone @usha-mandya PTAL if the wording looks good to you

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.12
+2021-12-12
+
+This release of Docker Engine contains changes in packaging only, and provides
+updates to the `docker scan` and `docker buildx` commands. Versions of `docker scan`
+before v0.11.0 are not able to [Log4j 2 CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228),
```
Comment (chris-crone) [inline engine/release-notes/index.md:30]:
```suggestion
before v0.11.0 are not able to detect the [Log4j 2 CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228),
```

**Review by chris-crone — APPROVED** [review]
LGTM

File: engine/release-notes/index.md (hunk 1 above)
Comment (thaJeztah) [reply engine/release-notes/index.md]: Ah, thx; updated 👍

**Review by chris-crone — APPROVED** [review]
LGTM

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
...
+$ yum install docker-scan-plugin
+```
+
+After upgrading, verify you have the latest version of `docker scan` installed:
+
+```console
+$ docker scan --accept-license --version
+Version:    v0.11.0
+Git commit: c8da19f
+Provider:   Snyk (1.563.0 (standalone))
```
Comment (thaJeztah) [inline engine/release-notes/index.md:63]: Note to self: need to update these tomorrow

File: engine/release-notes/index.md [hunk 3]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
...
+```
+
+[Read our blog post on CVE-2021-44228](https://www.docker.com/blog/apache-log4j-2-cve-2021-44228/)
+to learn how to use the `docker scan` command to check if images are vulnerable.
+
+## Packaging
+
+- Update `docker scan` to [v0.11.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.11.0).
```
Comment (thaJeztah) [inline engine/release-notes/index.md:71]: And here

File: engine/release-notes/index.md [hunk 4]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.12
+2021-12-13
+
+This release of Docker Engine contains changes in packaging only, and provides
+updates to the `docker scan` and `docker buildx` commands. Versions of `docker scan`
+before v0.11.0 are not able to detect the [Log4j 2 CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228),
+and we are shipping an updated version of `docker scan` in this release to help
```
Comment (usha-mandya) [inline engine/release-notes/index.md:31]:
```suggestion
We are shipping an updated version of `docker scan` in this release to help
```

File: engine/release-notes/index.md [hunk 5]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.12
+2021-12-13
+
+This release of Docker Engine contains changes in packaging only, and provides
+updates to the `docker scan` and `docker buildx` commands. Versions of `docker scan`
+before v0.11.0 are not able to detect the [Log4j 2 CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228),
```
Comment (usha-mandya) [inline engine/release-notes/index.md:30]:
```suggestion
before v0.11.0 are not able to detect the [Log4j 2 CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
```

File: engine/release-notes/index.md [hunk 6]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
...
+and we are shipping an updated version of `docker scan` in this release to help
+you scan your images for this vulnerability.
+
+> **Note**
+>
+> The `docker scan` command on Linux is currently only supported on x86 platforms.
+> We do not yet provide a package for other hardware architectures on Linux.
+
+The `docker scan` feature is provided as a separate package and depending on your
+upgrade- or installation method, may not be updated automatically to the latest
```
Comment (usha-mandya) [inline engine/release-notes/index.md:40]:
```suggestion
upgrade or installation method, may not be updated automatically to the latest
```

File: engine/release-notes/index.md [hunk 7]
```diff
@@ -22,6 +22,57 @@ for Docker Engine.
...
+version. Use the instructions below to update `docker scan` to the latest version.
+These instructions can also be used if you want to install, or upgrade the `docker scan`
```
Comment (usha-mandya) [inline engine/release-notes/index.md:42]:
```suggestion
You can also use these instructions to install, or upgrade the `docker scan`
```

**Review by jerae-duffin — APPROVED** [review]
LGTM!

**Pull request merged**